### PR TITLE
Modify subcameras positions in the arraycamera example

### PR DESCRIPTION
**Pull request by fernandojsg**
Just a slight modification on the array camera example to move each camera to make it obvious that each square represents a completely different camera not just duplicating the same one.

image

### CODE REVIEW

File: examples/webgl_camera_array.html [hunk 1]
```diff
@@ -36,13 +36,14 @@
...
 						var subcamera = new THREE.PerspectiveCamera( 40, ASPECT_RATIO, 0.1, 10 );
-						subcamera.bounds = new THREE.Vector4( x / AMOUNT, y / AMOUNT, SIZE, SIZE );
-						subcamera.position.x = ( x / AMOUNT ) - 0.5;
-						subcamera.position.y = 0.5 - ( y / AMOUNT );
-						subcamera.position.z = 1.5;
-						subcamera.position.multiplyScalar( 2 );
-						subcamera.lookAt( new THREE.Vector3() );
+						subcamera.bounds = new THREE.Vector4( xx, yy, SIZE, SIZE );
+						subcamera.position.x = ( (AMOUNT - 1) / 2 - x) * 0.2;
+						subcamera.position.y = ( (AMOUNT - 1) / 2 - y) * 0.2;
```
Comment (WestLangley) [inline examples/webgl_camera_array.html]: formatting: `subcamera.position.y = ( ( AMOUNT - 1 ) / 2 - y ) * 0.2;`

File: examples/webgl_camera_array.html [hunk 2]
```diff
@@ -36,13 +36,14 @@
...
+						subcamera.position.z = 3;
 						subcamera.updateMatrixWorld();
```
Comment (WestLangley) [inline examples/webgl_camera_array.html]: I see `updateMatrixWorld()` is required here... So the subcamera's world matrix is not updated by the renderer? Is that the desired behavior?

File: examples/webgl_camera_array.html (hunk 1 above)
Comment (fernandojsg) [inline examples/webgl_camera_array.html]: Thanks, fixed!

File: examples/webgl_camera_array.html (hunk 2 above)
Comment (fernandojsg) [inline examples/webgl_camera_array.html]: In the case of WebVR for example, you'll get the viewMatrix and the projectionMatrix for the camera from the API itself so you won't need that call.
I believe the typical use case for this camera array would be similar to the one with WebVR, you position your camera by yourself or by an external API and then just move the holder/rig that is what you'll get updated by the renderer.
@mrdoob could correct me if I'm wrong as he was the implementer of that part

Comment (fernandojsg) [inline examples/webgl_camera_array.html]: eg: https://github.com/mrdoob/three.js/blob/dev/examples/js/vr/WebVRCamera.js#L95-L108

**Comment by fernandojsg** [comment]
Nevermind I just saw that @mrdoob already did something similar https://github.com/mrdoob/three.js/commit/b31e0007c4be39a0855439ba28b05e81726c8186 O:)

File: examples/webgl_camera_array.html (hunk 2 above)
Comment (mrdoob) [inline examples/webgl_camera_array.html]: > I see `updateMatrixWorld()` is required here... So the subcamera's world matrix is not updated by the renderer? Is that the desired behavior?

That's something I'm still studying.

**Pull request closed without merging**